src/test/java/io/supertokens/test/HelloAPITest.java

```diff
@@ -337,15 +370,12 @@ public void testThatHelloAPIDoesNotRequireAPIKeys() throws Exception {
 
         String[] NOT_FOUND_ROUTES = new String[]{
                 "http://localhost:3567/abcd",
-                "http://localhost:3567/appid-hello", // app + /
-                "http://localhost:3567/appid-hello/", // app + /
-                "http://localhost:3567/appid-hello/test", // app + tenant + /
-                "http://localhost:3567/appid-hello/test/", // app + tenant + /
         };
 
         // Not found
         for (String notFoundUrl : NOT_FOUND_ROUTES) {
             try {
+                System.out.println(notFoundUrl);
```

Comment:
leftover println here

Reply:
Added it deliberately (in the test). Makes it easier to see which url the test failed on.